ezgui/src/canvas.rs

```diff
@@ -142,6 +146,24 @@ impl Canvas {
         } else if self.drag_just_ended {
             self.drag_just_ended = false;
         }
+
+        let cursor_screen_pt = self.get_cursor().to_pt();
+        let cursor_map_pt = self.screen_to_map(self.get_cursor());
+        let inner_bounds = self.get_inner_bounds();
+        let map_bounds = self.get_map_bounds();
+        if !inner_bounds.contains(cursor_screen_pt) && !self.disable_edge_panning
+           && map_bounds.contains(cursor_map_pt){
```

Comment:
Thanks for the explanation in the PR description! I'm still not sure it makes sense to auto-pan when the user is hovering over a UI element. For example, if I try to click the "more data" button in the top-right corner, auto-pan kicks in unexpectedly. There's generally enough open space along the screen edges to auto-pan, except for the OSD at the bottom. Nothing on there is clickable though, so maybe it's reasonable to remove `mark_covered_area` from `game/src/common/mod.rs`

Reply:
What about changing `PANNING_THRESHOLD` to 25? I've noticed that when I do that, no UI elements (as far as I can tell) are within the panning zone; thereby, preventing user attempts at interaction from causing unexpected panning.

Reply:
This seems fine, yeah